expression/chunk_executor.go

```diff
@@ -30,6 +30,34 @@ func Vectorizable(exprs []Expression) bool {
 			return false
 		}
 	}
+	return HasMoreThanOneSequenceFunction(exprs)
+}
+
+// HasMoreThanOneSequenceFunction indicates where the exprs can be evaluate as a vector.
+// When two or more of this three(nextval, lastval, setval) exists in exprs list and one of them is nextval, it should be eval row by row.
+func HasMoreThanOneSequenceFunction(exprs []Expression) bool {
```

Comment:
It seems this name doesn't conform to what it really checks.
It needs at least 2 functions.